tgui/packages/tgui/interfaces/Airlock.js

```diff
@@ -0,0 +1,110 @@
+import { useBackend } from '../backend';
+import { Box, Button, Flex, LabeledList, Section, Divider } from '../components';
+import { Window } from '../layouts';
+
+export const Airlock = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    signalers,
+    wireColors,
+    wireStates,
+    netId,
+    powerIsOn,
+    boltsAreUp,
+    aiControlDisabled,
+    safety,
+  } = data;
+
+  const handleWireInteract = (wireColorIndex, action) => {
+    act(action, { wireColorIndex });
+  };
+
+  const wires = Object.keys(wireColors);
+
+  return (
+    <Window
+      height={460}
+      width={370}>
+      <Window.Content>
+        <Section
+          title="Access panel">
+          <Box>
+            {"An identifier is engraved under the airlock's card sensors:"} <Box inline italic>{netId}</Box>
+          </Box>
+          <Divider />
+          <LabeledList>
+            { wires.map((entry, i) => (
+              <LabeledList.Item
+                key={entry}
+                label={(entry + " wire")}
+                labelColor={entry.toLowerCase()}>
+                {
+                  !wireStates[i]
+                    ? (
+                      <Box
+                        height={1.8} >
```

Comment:
Is explicit height setting here necessary? Do we even need this `Box` at all?

Reply:
I could it into an scss file too, but idk. The reason why I am doing it is to keep it from shrinking the height when something on the row changes.